File: middle-mile-delivery/client/src/utils/TripStatus.js
export const tripStatuses = [
    "PLANNED",
    "DEPARTED",
    "ARRIVED",
    "CANCELED"
]

export const tripStatusDisplayValue = status => {
    switch (status) {
        case "PLANNED":
            return "Planned";
        case "DEPARTED":
            return "Departed";
        case "ARRIVED":
            return "Arrived";
        case "CANCELED":
            return "Canceled";
        default:
            return "Invalid status";
    }
}

File: middle-mile-delivery/server/src/main/java/openerp/openerpresourceserver/utils/TripStatus.java
package openerp.openerpresourceserver.utils;

public enum TripStatus {
    PLANNED("PLANNED"),
    DEPARTED("DEPARTED"),
    ARRIVED("ARRIVED"),
    CANCELED("CANCELED");

    private final String code;

    TripStatus(String code) {
        this.code = code;
    }
    public String getCode() {
        return code;
    }
}
